### PR TITLE
Update readme with installation and usage info

### PR DESCRIPTION
**Pull request by i-like-robots**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,43 @@
...
+
+Word/s or partial word/s to filter tags with. E.G. chry will match the tags Chrysalis and Daimler Chrysler.
+
+**Required**
+
+### `count`
+
+The number of tags to return.
+
+Defaults to `10`.
```
Comment (andygout) [inline README.md]: Default is 20: https://github.com/Financial-Times/next-tme-search-api/blob/master/server/controllers/suggestions.js#L16

File: README.md [hunk 2]
```diff
@@ -1,6 +1,43 @@
...
+### Get search suggestions
+
+```sh
+curl https://next.ft.com/search-suggestions?q=china
+```
+
+### Limit number of results
+
+```sh
+curl https://next.ft.com/search-suggestions?q=china&count=5
```
Comment (andygout) [inline README.md]: Isn't it `limit=5` (converted to `count` in suggestions controller).

File: README.md [hunk 3]
```diff
@@ -1,6 +1,43 @@
...
+```
+
+## Parameters
+
+### `q`
+
+Word/s or partial word/s to filter tags with. E.G. chry will match the tags Chrysalis and Daimler Chrysler.
```
Comment (andygout) [inline README.md]: Maybe `Fiat Chrysler Automobiles` better example (in place of `Daimler Chrysler`) as existing data that is returned from query of `chry`?

**Comment by andygout** [comment]
👍  Bar comments.

File: README.md (hunk 2 above)
Comment (i-like-robots) [inline README.md]: so we do =S

**Pull request merged**